### DISCUSSION TITLE
extracting data from a paper

### DISCUSSION THREAD

**Discussion by rbuels** [GMOD/jbrowse-components#2634] [Ideas]
Pipe dream:

What if a researcher could see a paper that has some auxiliary data in it, and just kind of "open" the paper in JBrowse. Could we do a thing where we try to extract data links from a paper's supplemental materials?

**Comment by cmdcolin** [comment]
the data that is directly attached to papers via supp. data are generally badly formatted spreadsheets and such, rarely common bioinformatics formats we use.

one interesting thing that is in papers oftentimes are SRA ids but that requires complete pipelines to process and is not really suited for jbrowse itself

database curators would have experience with what is needed to go from paper->jbrowse
